packer/linux/conf/bin/bk-configure-cloudwatch-agent.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# shellcheck disable=SC2094
+set -euo pipefail
+
+if [[ "${CLOUDWATCH_ENABLE_METRICS:-false}" == "true" ]]; then
+    cw_config="/opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json"
+    cat <<<"$(jq \
+        --arg queue "$BUILDKITE_QUEUE" \
+        '. + {
+            metrics: {
+                metrics_collected: {
+                    mem: {measurement: ["mem_used_percent"], append_dimensions: {BuildkiteQueue: $queue}},
+                    disk: {measurement: ["used_percent"], resources: ["*"], append_dimensions: {BuildkiteQueue: $queue}}
```

Comment:
if we're adding the queue as dimension for memory and disk, should we do it for CPU too?